src/duckstation-qt/mainwindow.cpp

```diff
@@ -1628,7 +1623,17 @@ void MainWindow::clearGameListEntryPlayTime(const GameList::Entry* entry)
     return;
   }
 
-  GameList::ClearPlayedTimeForSerial(entry->serial);
+  if (!entry->IsDiscSet())
```

Comment:
It would be better to move this logic to `GameList`, that way it can be used by both Big Picture and the Qt UI.

Something like `void GameList::ClearPlayedTimeForEntry(const Entry* entry)`.